scripts/pre_commit_linter.py

```diff
@@ -1831,6 +1832,7 @@ def _lint_all_files(self):
             js_and_ts_messages.append(js_and_ts_stdout.get())
 
         python_utils.PRINT('')
+        # The source files are read as bytes, hence the b' prefix.
```

Comment:
This has nothing to do with source files right? Please update the comment. (The "source files" thing was for when you are actually reading the files in pylint_extensions.)

Reply:
Done